test/gerkin/features/steps/data_my_personal_data_usage.js

```diff
@@ -0,0 +1,126 @@
+const pactum = require('pactum');
+const { Given, When, Then, Before, After } = require('@cucumber/cucumber');
+const { header, localhost } = require('./helpers/helpers');
+
+let userID;
+let databaseID;
+let specDataMyPersonalDataUsage;
+
+const baseUrl = `${localhost}data/MyPersonalDataUsage/1.0`;
+
+Before(() => {
+  specDataMyPersonalDataUsage = pactum.spec();
+  databaseID = 'MCTS';
+});
+
+// Scenario: The user receives a list with all records that had read his personal data in the Digital Registries database
+Given(
+  'The user with id=282392302 wants to check who had read his personal data in the Digital Registries database',
+  () => {
+    userID = '282392302';
+    return userID;
+  }
+);
+
+Given(
+  'In the database, there is a list of users who read the user with id=282392302 personal data',
+  () => {
+    return 'In the database, there is a list of users who read his personal data';
+  }
+);
+
+When(
+  'The user with id=282392302 triggers an action to receive a list of users who read his personal data',
+  () => {
+    specDataMyPersonalDataUsage
+      .get(`${baseUrl}?userID=${userID}&DatabaseID=${databaseID}`)
```

Comment:
I see you use this URL build more than once in this file, can you move it to separate function? 
Also does pactum provide more convenient builders for URLs or is using template literals common approach?